python/adelphi/integration-tests/base-schema.cql

```diff
@@ -0,0 +1,211 @@
+CREATE KEYSPACE my_ks_0 WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'}  AND durable_writes = true;
```

Comment:
this is the base schema that gets executed in all C* versions

it's generated with: `python3 schema_util.py > ../../integration-tests/base-schema.cql`